docs/feeds.md

```diff
@@ -67,7 +67,7 @@ When the user creates a trigger from the CLI with the **--feed** parameter, the
 
 For example,assume the user has created a `mycloudant` binding for the `cloudant` package with their username and password as bound parameters. When the user issues the following command from the CLI:
 
-`wsk trigger create T --feed mycloudant/changes -p dbName myTable`
+`wsk trigger create T --feed mycloudant/changes --feed-param dbName myTable`
```

Comment:
Is there doc showing that `--feed-param` and `--trigger-param` can be used in conjunction with each other?

Reply:
It might be good to note that `-p` and `-P` only works for feed parameters.